releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h

```diff
@@ -1013,6 +1093,8 @@ struct FfiScanSummary *zcashlc_scan_blocks(const uint8_t *fs_block_cache_root,
                                            const uint8_t *db_data,
                                            uintptr_t db_data_len,
                                            int32_t from_height,
+                                           const uint8_t *from_state,
+                                           uintptr_t from_state_len,
```

Comment:
These new parameters have not been documented in the safety section. Non-blocking but please file an issue if it isn't fixed now.